src/timeunit.ts

```diff
@@ -336,10 +343,36 @@ export function formatExpression(timeUnit: TimeUnit, field: string, isUTCScale:
   }
 }
 
-export function normalizeTimeUnit(timeUnit: TimeUnit): TimeUnit {
+export function normalizeTimeUnit(timeUnit: TimeUnit | TimeUnitParams): TimeUnitParams {
+  if (!timeUnit) {
+    return undefined;
+  }
+
+  let params;
+  if (isString(timeUnit)) {
+    params = {
+      units: correctTimeUnit(timeUnit)
+    };
+  } else if (isObject(timeUnit)) {
+    params = {
+      ...timeUnit,
+      units: correctTimeUnit(timeUnit.units)
```

Comment:
`units` is an array, why is it working with the `correctTimeUnit` below.  

Some typing could be wrong here?

Reply:
units is a string